tests/unit/scatter-test.js

```diff
@@ -63,3 +63,38 @@ test('Margins are the right size when showLegend is no', function(assert) {
   assert.equal(component.get('marginRight'), 30, 'right margin exists');
   assert.equal(component.get('marginBottom'), 0, 'no bottom margin if showLegend is no');
 });
+
+/**
+ * Sets up `scatterContent` to be used in the domaing padding test.
+**/
+function setupDomainPaddingTest() {
+  scatterContent.graphicLeft = 40;
```

Comment:
Also, why doesn't this just return an object? Seems weird for it to set properties on some other object defined outside of the function